qiskit/chemistry/core/hamiltonian.py

```diff
@@ -321,10 +321,12 @@ def _process_z2symmetry_reduction(self, qubit_op, aux_ops):
                 z2_symmetries.tapering_values = self._z2symmetry_reduction
 
             logger.debug('Apply symmetry with tapering values %s', z2_symmetries.tapering_values)
-            z2_qubit_op = z2_symmetries.taper(qubit_op)
+            chop_to = 0.00000001  # Use same threshold as qubit mapping to chop tapered operator
```

Comment:
This would be more readable as `1e-8`

Reply:
I did it to be consistent with the threshold setting of the mapping in the FermionicOperator which is expressed the exact same way.

Reply:
Makes sense then. But, it should probably be changed everywhere then to use scientific notation. Counting zeros is tedious and error prone.

Reply:
The FermionicOperator needs refactoring around the qubit operator conversion to emit the new opflow instance types once the opflow Z2Symmetry support is added. This thresholding will need to be done in conjunction with the new opflow logic however that turns out. It may also make sense to expose this threshold on the core.Hamiltonian object so it can be adjusted at that level too by the user as desired. But yes I copy/pasted the value to make it the same though fortunately there are not too many zeros to count!

Reply:
Couldn't we just change it in both places to `1e-8`? This number just appears twice throughout Aqua, I think this would be legit to change instead of copying the less readable number. Also this is a minor change we can do right now, re-opening a new issue for this seems a bit too much overhead.